### DISCUSSION TITLE
Customizing all of Axum's response wording

### DISCUSSION THREAD

**Discussion by kellytk** [tokio-rs/axum#743] [Q&A] [answered]
Sometimes, such as with https://github.com/tokio-rs/axum/blob/main/axum/src/extract/typed_header.rs#L112, Axum communicates with the user using its own wording.

Are all instances of this enumerated somewhere?  How should they be customized?

**Comment by davidpdrsn** [comment, marked as answer]
https://docs.rs/axum/latest/axum/extract/index.html#customizing-extractor-responses

**Reply by kellytk** [reply]
Thank you.  To confirm, that's the only potential source of Axum's response wording to a user?

**Reply by davidpdrsn** [reply]
It should only come from extractors.
